.github/workflows/pre-commit.yml

```
@@ -1,12 +1,15 @@
-name: Run linters
+name: Run test commands
 
 on:
-  push:
-    branches: [ main ]
   pull_request:
     branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)